garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -314,13 +313,13 @@ local function GetKillerSample(victim, attacker, dmg)
    if IsValid(infl) and infl:IsNPC() then return end
 
    local dist = victim:GetPos():Distance(attacker:GetPos())
-   if dist > GetConVarNumber("ttt_killer_dna_range") then return nil end
+   if dist > GetConVar("ttt_killer_dna_range"):GetInt() then return nil end

```

Comment:
This is actually worse since `GetConVarNumber` is cached now. Either store the convar returned by `GetConVar` once, or just keep `GetConVarNumber`. The wiki page needs updating.